src/util/config-parser.h

```diff
@@ -0,0 +1,36 @@
+#ifndef CONFIG_PARSER_H
+#define CONFIG_PARSER_H
+
+#include <QIODevice>
+#include <QSettings>
+
+bool readConfig(QIODevice& device, QSettings::SettingsMap& map);
+bool writeConfig(QIODevice& device, const QSettings::SettingsMap& map);
+
+class MixxxSettings {
```

Comment:
IMO MixxxSettings is redundant since we're in the Mixxx repo -- either Settings or UserSettings may be a better name